lib/webauthn/signature_verifier.rb

```diff
@@ -39,6 +43,10 @@ def cose_algorithm
       end
     end
 
+    def rsa_pss?
+      cose_algorithm.name.start_with?("PS")
```

Comment:
This is a little quirky but would still work for PS384 and PS512 😄

Reply:
Yeah, fine for now.

Maybe, in the future, `COSE::Algorithm` can learn to respond to `#rsa_pss?` or something like that.